### PR TITLE
Accessibilities branch Pull Request

### PR DESCRIPTION
**Pull request by Peter-Dumbari**
# Port folio

in this PR:
- I added a new title https://github.com/Peter-Dumbari/Portofolio-Microverse/blob/9ea5e42f1c6d89a87690c5a2616e7b602d1bbd4b/index.html#L13.
- I added `alt` attribute to `img` tag 
https://github.com/Peter-Dumbari/Portofolio-Microverse/blob/9ea5e42f1c6d89a87690c5a2616e7b602d1bbd4b/index.html#L56-L68
- I renamed most of my `nav-links` https://github.com/Peter-Dumbari/Portofolio-Microverse/blob/9ea5e42f1c6d89a87690c5a2616e7b602d1bbd4b/index.html#L24-L27.
- I also added `label` to my form, making it invisible by using display none.
https://github.com/Peter-Dumbari/Portofolio-Microverse/blob/9ea5e42f1c6d89a87690c5a2616e7b602d1bbd4b/index.html#L309-L316

### CODE REVIEW

**Comment by TuyishimireEric** [comment]
# INVALID ❌ ❌ ❌
### Hi @Peter-Dumbari 👋

While you made a great effort in this project, I can't go ahead and review your code.

## Invalid Code Review Request ❌
Please, this is a **pair-programming project** 😨, Kindly make sure that everyone from your team has contributed equally to the project and submit again once you are done 🙏  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Kindly create  a new PR that follows the Gitflow approach

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

---
_Invalid Code Review Request does not count into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                     <li class="nav-link"><a href="#" class="link">About</a></li>
```
Comment (BregornOriginal) [inline index.html]: - Kindly as this is your portfolio, it should have a `title` tag like "your name" Portfolio. Or some meaningful for you than just the word Portfolio. In this way you can show a better title on your application 💯

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
## Hi @Team 

## Good job so far!

## There are some issues that you still need to work on to go to the next project but you are almost there!

one-step-closer-one-more

## Highlights

✅ Linters are passing
✅ Good commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joskalenda — APPROVED** [review]
| STATUS: APPROVED ✅ |
|------------------------|
| Hi @Team, 🥰|
| Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉  |
|  |

### Highlights
✅ ... Green linters
✅ ... Good fix accecibility

---------
### Optional suggestions

> _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### Cheers and Happy coding!👏👏👏
-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Peter-Dumbari** [comment]
# Acessibilities added
in this Merge:
- check the pull request for the full description

**Pull request merged**